examples/imagefactory/env.example

```diff
@@ -0,0 +1,11 @@
+EXTERNAL_IP=127.0.0.1
+
+SCHEMATICS_TALOS_VERSION=1.8.1
+SCHEMATICS_TALOS_ARCH=amd64
+SCHEMATICS_VALIDATE=false
+SCHEMATICS_SLEEP_TIME=600
+
+IMGFAC_VERSION=v0.5.0
+IMGFAC_EXTERNAL_URL=imgfac.example.com
```

Comment:
this should be a proper URL (when passed as an argument), this might be the reason the UI doesn't work properly, e.g. `http://imgfac.example.com/`

Reply:
I'd argue since no reverse-proxy is part of this scenario, we can assume it is always `http`.

See 9d937af721b9cb4393ab5bfb565af277efc5a5b0